sections/sprint-3/conclusion.tex

```diff
@@ -0,0 +1,64 @@
+\section{Sprint 3 conclusion}\label{sec:sprint3conclusion}
+This section concludes the preceding chapter on sprint 3.
+It will conclude the knowledge that was obtained based on the sections of the chapter, and discuss the retrospective meeting that was held in the end of the sprint.
```

Comment:
```suggestion
It will recapitulate the knowledge that was obtained based on the sections of the chapter, and discuss the retrospective meeting that was held in the end of the sprint.
```